x-pack/plugins/apm/public/components/shared/unified_search_bar/index.tsx

```diff
@@ -30,13 +32,27 @@ import { fromQuery, toQuery } from '../links/url_helpers';
 import { useApmParams } from '../../../hooks/use_apm_params';
 import { getBoolFilter } from '../get_bool_filter';
 import { useLegacyUrlParams } from '../../../context/url_params_context/use_url_params';
+import { clearCache } from '../../../services/rest/call_api';
+import { useTimeRangeId } from '../../../context/time_range_id/use_time_range_id';
+import {
+  toBoolean,
+  toNumber,
+} from '../../../context/url_params_context/helpers';
+
+export const DEFAULT_REFRESH_INTERVAL = 60000;
 
 function useSearchBarParams(defaultKuery?: string) {
   const { path, query } = useApmParams('/*');
   const urlKuery = 'kuery' in query ? query.kuery : undefined;
   const serviceName = 'serviceName' in path ? path.serviceName : undefined;
   const groupId = 'groupId' in path ? path.groupId : undefined;
   const environment = 'environment' in query ? query.environment : undefined;
+  const refreshIntervalFromUrl =
+    'refreshInterval' in query
+      ? toNumber(query.refreshInterval)
+      : DEFAULT_REFRESH_INTERVAL;
+  const refreshPausedFromUrl =
+    'refreshPaused' in query ? query.refreshPaused : 'true';
```

Comment:
I think these should be retrieved from advanced settings. @sqren?

Reply:
I'm not a huge fan of this approach tbh - we essentially lose any routing type safety. But out of scope for this PR.

Reply:
I saw they were retrieved from the Query Bar in the previous implementation of Date Picker, so i replicated them here.

https://github.com/achyutjhunjhunwala/kibana/blob/08925c46ae9241ec260903621cf21a6abe226f4a/x-pack/plugins/apm/public/components/shared/date_picker/apm_date_picker.tsx#L26-L31
https://github.com/elastic/kibana/blob/08925c46ae9241ec260903621cf21a6abe226f4a/x-pack/plugins/apm/public/components/shared/date_picker/apm_date_picker.tsx#L26-L31